evm_loader/program/src/account/ether_contract.rs

```diff
@@ -1,51 +1,11 @@
 use std::cell::RefMut;
 use std::mem::size_of;
 
-use arrayref::{array_ref, array_refs};
 use evm::{U256, Valids};
-use solana_program::pubkey::Pubkey;
 
-use crate::account::{EthereumAccount, Packable};
+use crate::account::EthereumAccount;
 use crate::config::STORAGE_ENTRIES_IN_CONTRACT_ACCOUNT;
 
-/// Ethereum contract data account v2

```

Comment:
Do not remove oldest versions of data structures. Sometimes we need to decode data stored in blockchain and it is hard to find their structure.